services/management/ia/templates_service.go

```diff
@@ -43,14 +43,16 @@ import (
 	"gopkg.in/reform.v1"
 	"gopkg.in/yaml.v3"
 
+	"github.com/percona/pmm-managed/data"
 	"github.com/percona/pmm-managed/models"
+	"github.com/percona/pmm-managed/utils/dir"
 )
 
 const (
-	builtinTemplatesPath = "/tmp/ia1/*.yml"
-	userTemplatesPath    = "/tmp/ia2/*.yml"
+	templatesDir = "/srv/ia/templates"
+	rulesDir     = "/etc/ia/rules"
 
-	ruleFileDir = "/tmp/ia1/"
+	dirPerm = os.FileMode(0o775)
 )
 
 // Template represents alerting rule template with added source field.
```

Comment:
We already have 3 other Template type definitions: models.Template, saas/pkg/alert.Template, and iav1beta1.Template. Please remove that one.

Reply:
saas/pkg/alert.Template doesn't have yaml and source fields I guess thats why @artemgavrilov created it here, which one should I reuse here? models.Template or iav1beta1.Template?

Reply:
> saas/pkg/alert.Template doesn't have yaml and source fields

The whole alert.Template describes that YAML

Reply:
my point being `alert.Template` doesn't have some of the fields (like source and yaml) that we need while returning all the collected templates in methods like `ListTemplates` so we can't reuse this type in place of the Template type defined in this file

That leaves us with 
- `iav1beta1.Template` which if we try to use will force us to convert `alert.Template` to `iav1beta1.Template` in `loadTemplatesFromAssets` and `loadTemplatesFromFiles` methods and various other places
- we have the same problems if we try to replace it with `models.Template`, it requires too many changes

so it doesn't seem like a quick change, I suggest that we skip it for now and remove this type some other time in some other ticket. @AlekSi let me know what you decide going offline now.